examples/next-sse-chat/src/server/db/schema.ts

```diff
@@ -1,8 +1,15 @@
 import type { InferSelectModel } from 'drizzle-orm';
 import { relations } from 'drizzle-orm';
 import { pgTableCreator, text, timestamp } from 'drizzle-orm/pg-core';
+import { railwayPrNumber } from '../railwayPrNumber';
 
-const pgTable = pgTableCreator((name) => `sse-chat_${name}`);
+const prPrefix = railwayPrNumber ? `__pr-${railwayPrNumber}_` : '';
+
+if (prPrefix) {
+  console.log('pull request prefix', prPrefix);
+}
+
+const pgTable = pgTableCreator((name) => `${prPrefix}sse-chat_${name}`);
```

Comment:
idea is to add a prefix for each PR environment so they're isolated

Reply:
note that ayou need to add this to drizzle config too under `schemaFilter` to make sure migration script also filters to use the prefix